de/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+  <title>Frag nicht zum Fragen, frag einfach</title>
+  <meta property="og:title" content="Frag nicht zum Fragen, frag einfach" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="de_DE" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Frag nicht zum Fragen, frag einfach</h1>
+    <p>
+      Seitdem ich im Internet unterwegs bin, gibt es immer irgendwelche
+      Leute, die sowas schreiben:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Irgendein Java-Experte hier?
+      </p>
+    </blockquote>
+    <p>
+      Das ist aus mehreren Gründen eine schlechte Form. Was die Person 
+      <em>eigentlcich</em> fragt ist:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Gibt es hier irgendeinen Java-Experten, der mir bei meinem Problem, was
+        auch immer es ist, selbst wenn es nichts mit Java zu tun, helfen könnte -
+        oder könnte mir wenigstens irgendjemand antworten?
+      </p>
+    </blockquote>
+    <p>
+      Es gibt sehr viele Gründe weshalb Menschen, die wirklich das Wissen haben,
+      es nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du für
+      mehr als du denkst.

```

Comment:
```suggestion
      Es gibt sehr viele Gründe weshalb Leute, die tatsächlich über die nötige Expertise verfügen,
      diese nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du nach
      mehr als du denkst.
```

I'm still not sure about the "fragst, um etwas zu fragen".

ask for -> fragen nach

"Wissen haben" -> imo something like "über Wissen verfügen" or "Expertise besitzen" is more proper to use, but ymmv.

Reply:
```suggestion
      Es gibt sehr viele Gründe weshalb Leute, die wirklich über das Wissen verfügen,
      es nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du nach
      mehr als du denkst.

```
Thanks. Imo, this sounds better though